whipper/command/cd.py

```diff
@@ -493,6 +493,10 @@ def _ripIfNotRipped(number):
                 continue
             _ripIfNotRipped(i + 1)
 
+        if self.options.cover_art == "embed" and self.coverArtPath is not None:
```

Comment:
Is the `and` condition needed?

Reply:
Hmmm `os.remove(None)` doesn't do anything right? .

Reply:
Also that makes logging a little weird when it logs `deleting file at : None`

Reply:
Sorry, forgot to consider the case in which no image is available (error, not in DB, etc.) :rofl:

Reply:
xD , no problem .